Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN OS=$TARGETOS ARCH=$TARGETARCH make $TARGETOS/$TARGETARCH
 FROM amazonlinux:2 AS linux-amazon
 RUN yum update -y
 RUN yum install util-linux libyaml -y \
-    && amazon-linux-extras install -y lustre2.10
+    && amazon-linux-extras install -y lustre
```

Comment:
Hey, question, why aren't we version pinning here anymore?

Reply:
We chose to move to a versionless repo to easier facilitate upgrades for customers. This helps protect against the scenario where customer pins an older version in some configuration file and forgets about it.

Reply:
This seems like it risks issues with unintentional upgrades. Please, reconsider offering a pinned version here (or at least tag with the built Dockerfile version in the official ECR images)